package.json

```diff
@@ -31,32 +31,39 @@
   "homepage": "https://github.com/webdriverio/wdio-allure-reporter#readme",
   "dependencies": {
     "allure-js-commons": "^1.2.0",
-    "babel-runtime": "^5.8.25"
+    "babel-runtime": "^6.26.0"
   },
   "devDependencies": {
-    "babel": "^5.8.23",
-    "babel-core": "^5.8.23",
-    "babel-eslint": "^4.1.7",
-    "chai": "^3.5.0",
-    "cheerio": "^0.20.0",
-    "del": "^2.2.0",
-    "eslint": "^1.4.1",
-    "eslint-config-standard": "^4.3.2",
-    "eslint-plugin-standard": "^1.3.0",
-    "grunt": "^0.4.5",
-    "grunt-babel": "^5.0.1",
-    "grunt-bump": "^0.6.0",
-    "grunt-cli": "^0.1.13",
-    "grunt-contrib-clean": "^0.6.0",
+    "babel-cli": "~6.26.0",
+    "babel-core": "~6.25.0",
+    "babel-eslint": "~8.1.0",
+    "babel-plugin-add-module-exports": "~0.2.1",
+    "babel-plugin-transform-runtime": "~6.23.0",
+    "babel-preset-es2015": "~6.24.1",
+    "babel-preset-stage-0": "~6.24.1",
+    "chai": "^4.1.2",
+    "cheerio": "^0.22.0",
+    "del": "^3.0.0",
+    "eslint": "^4.14.0",
+    "eslint-config-standard": "^11.0.0-beta.0",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-node": "^5.2.1",
+    "eslint-plugin-promise": "^3.6.0",
+    "eslint-plugin-standard": "^3.0.1",
+    "grunt": "^1.0.1",
+    "grunt-babel": "^7.0.0",
+    "grunt-bump": "^0.8.0",
+    "grunt-cli": "^1.2.0",
+    "grunt-contrib-clean": "^1.1.0",
     "grunt-contrib-connect": "^1.0.2",
-    "grunt-contrib-watch": "^0.6.1",
-    "grunt-eslint": "^17.1.0",
-    "grunt-mocha": "^1.0.2",
-    "grunt-mocha-test": "^0.12.7",
-    "grunt-npm": "0.0.2",
-    "load-grunt-tasks": "^3.2.0",
-    "wdio-mocha-framework": "^0.3.5",
-    "webdriverio": "^4.1.1"
+    "grunt-contrib-watch": "^1.0.0",
+    "grunt-eslint": "^20.1.0",
+    "grunt-mocha": "^1.0.4",
+    "grunt-mocha-test": "^0.13.3",
+    "grunt-npm": "^0.0.2",
+    "load-grunt-tasks": "^3.5.2",
+    "wdio-mocha-framework": "^0.5.12",
+    "webdriverio": "4.8.0"
```

Comment:
Here I use old `webdriverio` version because in the newest one I ran into issue https://github.com/webdriverio/webdriverio/issues/2494 in test https://github.com/webdriverio/wdio-allure-reporter/blob/master/test/specs/after.js